src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService serv = new UserService();
```

Comment:
don't shorten your local variables' names, be verbose! 🙏  Remember, developers spend much more time reading the code of others than writing their own, time saved by not typing out a few extra letters is not worth it